platformio.ini

```diff
@@ -686,42 +672,29 @@ debug_init_break  =
 #
 [env:BIGTREE_GTR_V1_0]
 platform          = ststm32@>=5.7.0,<6.2.0
```

Comment:
If the `>=5.7.0` is not important here, then this can also be `${common_stm32.platform}`.

Reply:
They were all intended to use the common platform version. There were probably some merge issues I imagine.